ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/PartitionExport.java

```diff
@@ -75,64 +77,62 @@ class PartitionExport {
+    exportService.submit(new ExportJob() {
+      @Override
+      public void run() {
+        try {
+          write(forReplicationSpec, isExportTask, fileList, dataCopyAtLoad);
+        } catch(InterruptedException e) {
+          throw new RuntimeException(e.getCause().getMessage(), e);
+        } catch (HiveException e) {
+          throw new RuntimeException(e.getCause().getMessage(), e);
+        }
+      }
+     }
+     );
+  }
+  private List<DataCopyPath> write(final ReplicationSpec forReplicationSpec, boolean isExportTask,
                                    FileList fileList, boolean dataCopyAtLoad)
           throws InterruptedException, HiveException {
-    List<Future<?>> futures = new LinkedList<>();
     List<DataCopyPath> managedTableCopyPaths = new LinkedList<>();
+    List<Future<?>> futures = new LinkedList<>();
     ExecutorService producer = Executors.newFixedThreadPool(1,
-        new ThreadFactoryBuilder().setNameFormat("partition-submitter-thread-%d").build());
+            new ThreadFactoryBuilder().setNameFormat("partition-submitter-thread-%d").build());
     futures.add(producer.submit(() -> {
       SessionState.setCurrentSessionState(callersSession);
       for (Partition partition : partitionIterable) {
         try {
           queue.put(partition);
         } catch (InterruptedException e) {
           throw new RuntimeException(
-              "Error while queuing up the partitions for export of data files", e);
+                  "Error while queuing up the partitions for export of data files", e);
         }
       }
     }));
     producer.shutdown();
 
     ThreadFactory namingThreadFactory =
-        new ThreadFactoryBuilder().setNameFormat("partition-dump-thread-%d").build();
+            new ThreadFactoryBuilder().setNameFormat("partition-dump-thread-%d").build();
     ExecutorService consumer = Executors.newFixedThreadPool(nThreads, namingThreadFactory);
 
     while (!producer.isTerminated() || !queue.isEmpty()) {
-      /*
-      This is removed using a poll because there can be a case where there partitions iterator is empty
-      but because both the producer and consumer are started simultaneously the while loop will execute
-      because producer is not terminated but it wont produce anything so queue will be empty and then we
-      should only wait for a specific time before continuing, as the next loop cycle will fail.
-       */
+    /*
+    This is removed using a poll because there can be a case where there partitions iterator is empty
+    but because both the producer and consumer are started simultaneously the while loop will execute
+    because producer is not terminated but it wont produce anything so queue will be empty and then we
+    should only wait for a specific time before continuing, as the next loop cycle will fail.
+     */
       Partition partition = queue.poll(1, TimeUnit.SECONDS);
```

Comment:
Please respect the original indentation. Sometimes your indentation is better. But other committers may have a different opinion. We don't want to make repetitive changes to grow. Also it's much cleaner on patch files, too.